src/Sentry/Internal/Extensions/MiscExtensions.cs

```diff
@@ -82,4 +82,19 @@ public static void CancelAfterSafe(this CancellationTokenSource cts, TimeSpan ti
         TKey key,
         TValue value) =>
         collection.Add(new KeyValuePair<TKey, TValue>(key, value));
+
+    internal static string GetRawMessage(this AggregateException exception)
```

Comment:
Are there any concerns with this assuming there's only one level nested exception?

For example, if an exception `B` nested in exception `A` and `A` is used to throw an `AggregateException`, the returned result will be neither `B`'s nor `A`'s message, but rather `"One or more errors occurred."`

Reply:
"One or more errors occurred." is actually what we're after here.  Rather than "One or more errors occurred. (error 1) (error 2)"

Only reason not to take just the default message is that technically it can be customized.  Even then, we'd just want the custom message, before the inner exceptions messages were appended.

Reply:
In that case please ignore my comment 🙂